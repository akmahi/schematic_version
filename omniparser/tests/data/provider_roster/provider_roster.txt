ACO|MEGEN|CR0391|20220517||HEC|POPHLTHMGMT-HEC
PDAPR|010324279|Atlee Gleaton Eye Care|1932284171|||1|P|1821064023||10903711||Schumacher-Feero||Linda|MD||207-622-3185||18|Ophthalmology|207W00000X||||SPC|||||||||||||||||1|EBPA|||Kennebec Region Health Alliance|08181999||Active|Y||||||
PDAPR|010324279|Atlee Gleaton Eye Care|1932284171|||1|P|1821064023||10903711||Schumacher-Feero||Linda|MD||207-622-3185||18|Ophthalmology|207W00000X||||SPC|||||||||||||||||1|Soarian|||Kennebec Region Health Alliance|08181999||Active|Y||||||
PDAPR|472155341|Family Focused Healthcare|1568867885|||1|P|1740522648||||Trask||Monica|PA||207-872-4303||97|Family Medicine|363A00000X||||PCP|||||||||||||||||1|EBPA|||Kennebec Region Health Alliance|02192014||Active|Y||||||
PDAPR|472155341|Family Focused Healthcare|1568867885|||1|P|1740522648||||Trask||Monica|PA||207-872-4303||97|Family Medicine|363A00000X||||PCP|||||||||||||||||1|Soarian|||Kennebec Region Health Alliance|02192014||Active|Y||||||
PDAPR|010316462|Eye Care of Maine|1710953336|||1|P|1790772887||10904076||Witkin||Steve|MD||207-873-2731||18|Ophthalmology|207W00000X||||SPC|||||||||||||||||1|EBPA|||Kennebec Region Health Alliance|05061998||Active|Y||||||
PDAPR|010316462|Eye Care of Maine|1710953336|||1|P|1790772887||10904076||Witkin||Steve|MD||207-873-2731||18|Ophthalmology|207W00000X||||SPC|||||||||||||||||1|Soarian|||Kennebec Region Health Alliance|05061998||Active|Y||||||
PDAPR|010318691|Kennebec Anesthesia Associates (A)|1992789184|||1|P|1093906786||12273992||Wyse||Jason|MD||207-620-1136|Jason.Wyse@MaineGeneral.org|5|Anesthesiology|207L00000X||||SPC|||||||||||||||||1|EBPA|||Kennebec Region Health Alliance|08172011||Active|Y||||||
PDAPR|010318691|Kennebec Anesthesia Associates (A)|1992789184|||1|P|1093906786||12273992||Wyse||Jason|MD||207-620-1136|Jason.Wyse@MaineGeneral.org|5|Anesthesiology|207L00000X||||SPC|||||||||||||||||1|Soarian|||Kennebec Region Health Alliance|08172011||Active|Y||||||
PDAPR|010350921|Pathology Associates PA|1346300738|||1|P|1770704728||12518868||Nelsen||Laura|MD||207-626-1409|Laura.Nelsen@MaineGeneral.org|22|Pathology|207ZP0102X||||SPC|||||||||||||||||1|EBPA|||Kennebec Region Health Alliance|11202013||Active|Y||||||
PDAPR|010350921|Pathology Associates PA|1346300738|||1|P|1770704728||12518868||Nelsen||Laura|MD||207-626-1409|Laura.Nelsen@MaineGeneral.org|22|Pathology|207ZP0102X||||SPC|||||||||||||||||1|Soarian|||Kennebec Region Health Alliance|11202013||Active|Y||||||