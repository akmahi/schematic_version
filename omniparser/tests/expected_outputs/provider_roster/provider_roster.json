[
	{
		"provider": {
			"aco_effective_date": "08181999",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"external_id": "10903711",
			"first_name": "Linda",
			"insurance_plan": {
				"company_name": "EBPA"
			},
			"last_name": "Schumacher-Feero",
			"name_suffix": "MD",
			"npi": "1821064023",
			"pcp_flag": "SPC",
			"phone_work": "207-622-3185",
			"practice": {
				"group_npi": "1932284171",
				"location_rank": "1",
				"name": "Atlee Gleaton Eye Care",
				"tin": "010324279"
			},
			"specialty_code": "18",
			"taxonomy_code": "207W00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "08181999",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"external_id": "10903711",
			"first_name": "Linda",
			"insurance_plan": {
				"company_name": "Soarian"
			},
			"last_name": "Schumacher-Feero",
			"name_suffix": "MD",
			"npi": "1821064023",
			"pcp_flag": "SPC",
			"phone_work": "207-622-3185",
			"practice": {
				"group_npi": "1932284171",
				"location_rank": "1",
				"name": "Atlee Gleaton Eye Care",
				"tin": "010324279"
			},
			"specialty_code": "18",
			"taxonomy_code": "207W00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "02192014",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"first_name": "Monica",
			"insurance_plan": {
				"company_name": "EBPA"
			},
			"last_name": "Trask",
			"name_suffix": "PA",
			"npi": "1740522648",
			"pcp_flag": "PCP",
			"phone_work": "207-872-4303",
			"practice": {
				"group_npi": "1568867885",
				"location_rank": "1",
				"name": "Family Focused Healthcare",
				"tin": "472155341"
			},
			"specialty_code": "97",
			"taxonomy_code": "363A00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "02192014",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"first_name": "Monica",
			"insurance_plan": {
				"company_name": "Soarian"
			},
			"last_name": "Trask",
			"name_suffix": "PA",
			"npi": "1740522648",
			"pcp_flag": "PCP",
			"phone_work": "207-872-4303",
			"practice": {
				"group_npi": "1568867885",
				"location_rank": "1",
				"name": "Family Focused Healthcare",
				"tin": "472155341"
			},
			"specialty_code": "97",
			"taxonomy_code": "363A00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "05061998",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"external_id": "10904076",
			"first_name": "Steve",
			"insurance_plan": {
				"company_name": "EBPA"
			},
			"last_name": "Witkin",
			"name_suffix": "MD",
			"npi": "1790772887",
			"pcp_flag": "SPC",
			"phone_work": "207-873-2731",
			"practice": {
				"group_npi": "1710953336",
				"location_rank": "1",
				"name": "Eye Care of Maine",
				"tin": "010316462"
			},
			"specialty_code": "18",
			"taxonomy_code": "207W00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "05061998",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"external_id": "10904076",
			"first_name": "Steve",
			"insurance_plan": {
				"company_name": "Soarian"
			},
			"last_name": "Witkin",
			"name_suffix": "MD",
			"npi": "1790772887",
			"pcp_flag": "SPC",
			"phone_work": "207-873-2731",
			"practice": {
				"group_npi": "1710953336",
				"location_rank": "1",
				"name": "Eye Care of Maine",
				"tin": "010316462"
			},
			"specialty_code": "18",
			"taxonomy_code": "207W00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "08172011",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"email": "Jason.Wyse@MaineGeneral.org",
			"external_id": "12273992",
			"first_name": "Jason",
			"insurance_plan": {
				"company_name": "EBPA"
			},
			"last_name": "Wyse",
			"name_suffix": "MD",
			"npi": "1093906786",
			"pcp_flag": "SPC",
			"phone_work": "207-620-1136",
			"practice": {
				"group_npi": "1992789184",
				"location_rank": "1",
				"name": "Kennebec Anesthesia Associates (A)",
				"tin": "010318691"
			},
			"specialty_code": "5",
			"taxonomy_code": "207L00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "08172011",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"email": "Jason.Wyse@MaineGeneral.org",
			"external_id": "12273992",
			"first_name": "Jason",
			"insurance_plan": {
				"company_name": "Soarian"
			},
			"last_name": "Wyse",
			"name_suffix": "MD",
			"npi": "1093906786",
			"pcp_flag": "SPC",
			"phone_work": "207-620-1136",
			"practice": {
				"group_npi": "1992789184",
				"location_rank": "1",
				"name": "Kennebec Anesthesia Associates (A)",
				"tin": "010318691"
			},
			"specialty_code": "5",
			"taxonomy_code": "207L00000X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "11202013",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"email": "Laura.Nelsen@MaineGeneral.org",
			"external_id": "12518868",
			"first_name": "Laura",
			"insurance_plan": {
				"company_name": "EBPA"
			},
			"last_name": "Nelsen",
			"name_suffix": "MD",
			"npi": "1770704728",
			"pcp_flag": "SPC",
			"phone_work": "207-626-1409",
			"practice": {
				"group_npi": "1346300738",
				"location_rank": "1",
				"name": "Pathology Associates PA",
				"tin": "010350921"
			},
			"specialty_code": "22",
			"taxonomy_code": "207ZP0102X",
			"work_location_priority": "P"
		}
	},
	{
		"provider": {
			"aco_effective_date": "11202013",
			"aco_network_name": "Kennebec Region Health Alliance",
			"aco_provider_status": "Active",
			"aco_signed_contract": "Y",
			"email": "Laura.Nelsen@MaineGeneral.org",
			"external_id": "12518868",
			"first_name": "Laura",
			"insurance_plan": {
				"company_name": "Soarian"
			},
			"last_name": "Nelsen",
			"name_suffix": "MD",
			"npi": "1770704728",
			"pcp_flag": "SPC",
			"phone_work": "207-626-1409",
			"practice": {
				"group_npi": "1346300738",
				"location_rank": "1",
				"name": "Pathology Associates PA",
				"tin": "010350921"
			},
			"specialty_code": "22",
			"taxonomy_code": "207ZP0102X",
			"work_location_priority": "P"
		}
	}
]
